client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,16 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
```

Comment:
@rickybanister Ah, sorry, the git diff was a bit misleading I guess. Do you know why `.is-link` is deprecated in the first place? I've been chatting a bit with @tyxla and I suspected it might be the case that if something should _appear_ as a link, it should be _rendered_ as a link (i.e. `<a />` element).